tests/gocase/unit/type/hash/hash_test.go

```diff
@@ -783,6 +783,53 @@ func TestHash(t *testing.T) {
 
 			require.Equal(t, []interface{}{"field1", "some-value", "field2", ""}, rdb.Do(ctx, "HrangeByLex", testKey, "[a", "[z").Val())
 		})
+
+		t.Run("HRandField count is positive", func(t *testing.T) {
+			testKey := "test-hash-1"
+			require.NoError(t, rdb.Del(ctx, testKey).Err())
+			require.NoError(t, rdb.HSet(ctx, testKey, "key1", "value1", "key2", "value2", "key3", "value3").Err())
+			result1, err1 := rdb.HRandField(ctx, testKey, 5).Result()
```

Comment:
can you use same err and result to avoid too many variable names?

Reply:
Ok, is there anything else that needs to be improved?

Reply:
I guess no, if these are finished I'll approve this pull request. Thanks for your working